fpdf/fpdf.py

```diff
@@ -1483,9 +1483,13 @@ def draw_path(self, path, debug_stream=None):
             debug_stream (TextIO): print a pretty tree of all items to be rendered
                 to the provided stream. To store the output in a string, use
                 `io.StringIO`.
+            _copy (bool): if true (the default), the path will be copied before being
```

Comment:
Could you please remove the leading udnerscore as this is a public API parameter?